kin/client.py

```diff
@@ -0,0 +1,370 @@
+        :raises: :class:`KinErrors.AccountNotFoundError`: if the account does not exist.
+        """
+        # TODO: might want to simplify the returning data
+        if not is_valid_address(address):
+            raise ValueError('invalid address: {}'.format(address))
+
+        try:
+            acc = self.horizon.account(address)
+            return AccountData(acc, strict=False)
+        except Exception as e:
+            err = KinErrors.translate_error(e)
+            raise KinErrors.AccountNotFoundError(address) if \
+                isinstance(err, KinErrors.ResourceNotFoundError) else err
+
+    def get_transaction_data(self, tx_hash, simple=True):
+        """Gets transaction data.
+
+        :param str tx_hash: transaction hash.
+        :param boolean simple: (optional) returns a simplified transaction object
+
+        :return: transaction data
+        :rtype: :class:`kin.TransactionData` or `kin.SimplifiedTransaction`
+
+        :raises: ValueError: if the provided hash is invalid.
+        :raises: :class:`KinErrors.ResourceNotFoundError`: if the transaction does not exist.
+        :raises: :class:`KinErrors.CantSimplifyError`: if the tx is too complex to simplify
+        """
+        if not is_valid_transaction_hash(tx_hash):
+            raise ValueError('invalid transaction hash: {}'.format(tx_hash))
+
+        try:
+            tx = self.horizon.transaction(tx_hash)
+
+            # get transaction operations
+            tx_ops = self.horizon.transaction_operations(tx['hash'], params={'limit': 100})
+            tx['operations'] = tx_ops['_embedded']['records']
+
+            tx_data = TransactionData(tx, strict=False)
+        except Exception as e:
+            raise KinErrors.translate_error(e)
+
+        if simple:
+            return SimplifiedTransaction(tx_data, self.kin_asset)
+        return tx_data
+
+    def verify_kin_payment(self, tx_hash, source, destination, amount, memo=None, check_memo=False):
+        """
+        Verify that a give tx matches the desired parameters
+        :param str tx_hash: The hash of the transaction to query
+        :param str source: The expected source account
+        :param str destination: The expected destination account
+        :param float amount: The expected amount
+        :param str memo: (optional) The expected memo
+        :param boolean check_memo: (optional) Should the memo match
+        :return: True/False
+        :rtype: boolean
+        """
+
+        tx = self.get_transaction_data(tx_hash)
+        operation = tx.operation
+        if operation.type != OperationTypes.PAYMENT:
+            return False
+        if operation.asset != self.kin_asset.code:
+            return False
+        if source != tx.source or destination != operation.destination or amount != operation.amount:
+            return False
+        if check_memo and memo != tx.memo:
+            return False
+
+        return True
+
+    def friendbot(self, address):
```

Comment:
I don't like much the name, non blockchain oriented users don't know what's friendbot, the boring, obvious name of "create_and_fund" will be much better for SDK